sdks/python/apache_beam/ml/transforms/tft_test.py

```diff
@@ -130,7 +125,6 @@ def test_ScaleTo01_list(self):
           | "MLTransform" >> base.MLTransform(
               write_artifact_location=self.artifact_location).with_transform(
                   tft.ScaleTo01(columns=['x'])))
-      _ = (list_result | beam.Map(assert_ScaleTo01_artifacts))
```

Comment:
Rather than removing all of these checks, can we check that the written artifact is correct?

Reply:
I added #29017 to do this but for now, we will have to use some TF tools to do it. It is not clear how to do it but it is not straight forward. 

To read the artifacts, we use `read_artifact_location` and there are tests covering this path.